### PR TITLE
Slightly improved docker images

### PR DESCRIPTION
**Pull request by jamieparkinson**
This is a start at decoupling the apps in order to make builds faster and to make this repo a little less of a monolith - it moves us closer to being able to make the changes in https://github.com/wellcomecollection/wellcomecollection.org/issues/7038, but shouldn't actually change very much as-is (although it does also fix the bundle analysis file naming and deployment).

### CODE REVIEW

File: common/next/next.config.js [hunk 1]
```diff
@@ -7,102 +7,103 @@ const withMDX = require('@next/mdx')({
...
-                  analyzerMode: 'static',
-                  generateStatsFile: true,
-                  openAnalyzer: false,
-                  statsFilename: `${bundleAnalysisFile}.json`,
-                  reportFilename: `${bundleAnalysisFile}.html`,
-                })
-              );
-            }
+          config.module.rules.push({
+            test: /\.css$/,
```
Comment (davidpmccormick) [inline common/next/next.config.js:96]: do we need these still?

File: common/package.json [hunk 2]
```diff
@@ -28,6 +28,7 @@
     "@types/uuid": "^8.3.0",
     "@weco/toggles": "1.0.0",
     "@wojtekmaj/enzyme-adapter-react-17": "^0.6.3",
+    "axios": "^0.21.4",
```
Comment (davidpmccormick) [inline common/package.json:31]: Was this added for something in particular? I prefer `axios`, but we've tended to use (the horrifically titled) `isomorphic-unfetch` for most things

File: common/next/next.config.js (hunk 1 above)
Comment (jamieparkinson) [reply common/next/next.config.js:96]: Good question, I just copied them over in https://github.com/wellcomecollection/wellcomecollection.org/pull/7032. I'll try removing them in a separate PR as this contains an unnerving amount of build changes already

File: common/package.json (hunk 2 above)
Comment (jamieparkinson) [reply common/package.json:31]: It was being used by a component in common (I think user info related) but wasn't installed as a dep (I think I forgot to add the dep when I extracted said component from the identity app) so caused the build to fail when it was separated out more strictly. Indeed we should use one for everything, I'll do that separately as well.

Comment (davidpmccormick) [reply common/package.json:31]: if the one thing for everything could be `axios` that'd be nice… (probably too much of a rewrite of too many things at this point, though).

**Review by davidpmccormick — APPROVED** [review]

File: common/package.json (hunk 2 above)
Comment (jamieparkinson) [reply common/package.json:31]: Interesting, looks like Next [now provides a polyfill for fetch](https://nextjs.org/docs/basic-features/supported-browsers-features#server-side-polyfills)

**Pull request merged**